library/Vanilla/Theme/ThemeProviderInterface.php

```diff
@@ -100,9 +100,10 @@ public function setPreviewTheme($themeID): array;
     /**
      * Get "current" theme.
      *
+     * @param int/string $themeID
      * @return array
      */
-    public function getCurrent(): ?array;
+    public function getCurrent($themeID): ?array;
```

Comment:
I'm not really understanding why we've co-opted this. Don't we already have `getThemeWithAssets()`?